apps/src/templates/sectionAssessments/FreeResponsesAssessments.story.jsx

```diff
@@ -12,20 +12,19 @@ const questionOne = [
     id: 2,
     studentId: '211',
     name: 'Maddie',
-    response: `As trees get older they lose their chlorophyll. In painting, you have unlimited power.
-        You have the ability to move mountains. You can bend rivers. But when I get home, the only thing
-        I have power over is the garbage..,`
+    response: `Trees live in your fan brush, but you have to scare them out. Use absolutely no pressure. Just like an angel's wing. You can't have light without dark. You can't know happiness unless you've known sorrow. If you didn't have baby clouds, you wouldn't have big clouds. It is a lot of fun. We'll put all the little clouds in and let them dance around and have fun. And right there you got an almighty cloud. We don't have to be committed. We are just playing here. A tree cannot be straight if it has a crooked trunk. Let your heart take you to wherever you want to be. No worries. No cares. Just float and wait for the wind to blow you around. Put light against light - you have nothing. Put dark against dark - you have nothing. It's the contrast of light and dark that each give the other one meaning. This is truly an almighty mountain. The only thing worse than yellow snow is green snow. Paint anything you want on the canvas. Create your own world. You don't have to be crazy to do this but it does help. Tree trunks grow however makes them happy. Now let's put some happy little clouds in here. In your imagination you can go anywhere you want. It's so important to do something every day that will make you happy. Almost everything is going to happen for you automatically - you don't have to spend any time working or worrying. I'm a water fanatic. I love water.
```

Comment:
I thought our stance was to break up long lines, but it might be different since this is a string? Might be good to take a look at the style guide.

Reply:
I checked Code.org style guide but did not find information specific to strings.  Good news! Airbnb style guide does provide some guidance (see below):

" 6.2 Strings that cause the line to go over 100 characters should not be written across multiple lines using string concatenation. "